packages/menus/src/index.js

```diff
@@ -0,0 +1,2 @@
+export * from './utils';
```

Comment:
Minor note: I tend to think of utils as a junk drawer for naming. Since we have a whole new package here, what might be more descriptive to readers and a better fit? 

What the logic currently does is related to transformation/serialization. What other type of things might we add to this package?

Reply:
Removed.